README.rst

```diff
@@ -134,3 +134,5 @@ Read `Contributing <http://transcriptic.readthedocs.io/en/latest/contributing.ht
    :target: http://transcriptic.readthedocs.io/en/latest/?badge=latest
 .. |Code Coverage| image:: https://codecov.io/gh/strateos/transcriptic/branch/master/graph/badge.svg
    :target: https://codecov.io/gh/strateos/transcriptic
+.. |Binder| image:: https://mybinder.org/badge_logo.svg
+  :target: https://mybinder.org/v2/gh/strateos/transcriptic/master
```

Comment:
try with 
 https://mybinder.org/v2/gh/strateos/transcriptic/SEA-562_add-binder
to see what will be rendered once merged

Reply:
Does the uploaded version not have the mocked api connection changes?
![image](https://user-images.githubusercontent.com/54077638/101517919-23687e80-3936-11eb-80a7-963949750f39.png)
This is pretty cool though - my first time learning about this. It's mainly for Python and other languages offering a REPL?

Reply:
Yea.. the base image uses the latest version of txpy but we haven't released the latest changes yet.

Yup! Its pretty common in the scientific community. The current demo notebook isn't the best example.. its commonly used in conjunction with plotting libs and other visualizations